MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -1628,8 +2190,15 @@ public static Person generateInstanceFromXML(Node wn, Campaign c, Version versio
             for (int x = 0; x < nl.getLength(); x++) {
                 Node wn2 = nl.item(x);
 
-                if (wn2.getNodeName().equalsIgnoreCase("name")) {
-                    retVal.name = wn2.getTextContent();
+                // TODO : reorder these based on what is most likely to show up, to improve load speed
```

Comment:
I'm not so sure that'll help. I went through and rewrote this to use `equals` (as case is important in XML) and also a trie, the node name matching did not end up being a significant part of the work.

Reply:
Just one of the performance ideas I was working on, I also found it didn't amount to much, but forgot to remove. Removed.